src/main/java/mate/academy/hwspringbootintro/dto/auth/RegisterUserRequestDto.java

```diff
@@ -0,0 +1,27 @@
+package mate.academy.hwspringbootintro.dto.auth;
+
+import jakarta.validation.constraints.NotNull;
+import jakarta.validation.constraints.Size;
+import lombok.Getter;
+import lombok.Setter;
+import mate.academy.hwspringbootintro.validator.annotation.FieldMatch;
+
+@Getter
+@Setter
+@FieldMatch(first = "password", second = "repeatPassword", message = "Given passwords has to "
+        + "match each other. Please check the input")
+public class RegisterUserRequestDto {
+    @NotNull
```

Comment:
for String validation use `@NotBlank`
```suggestion
    @NotBlank
```